doc/README.md

```diff
@@ -1,6 +1,5 @@
 # Getting Started
 
-
```

Comment:
These empty lines that you have removed will make the markdown linter fail (there should be two empty lines before every section).